misc/find_prime_numbers.c

```diff
@@ -0,0 +1,134 @@
+/*
+This program will find prime numbers upto 2,000,000,000 (2 Billion)
+Here we have option to either
+1. print all prime numbers in console,
+2. show total number of prime exist till given number
+3. or create a txt file and write prime numbers in that txt file
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>     /// for IO operations
#include <assert.h>    /// for assert
```